### DISCUSSION TITLE
Z-Wave LXC update to 9.14.2

### DISCUSSION THREAD

**Discussion by grzech1983** [tteck/Proxmox#3326] [General]
Hello,

can I ask for update for Z-Wave LXC to 9.14.2 version? This is needed by latest Home Assistant 2024.7.0:

https://www.home-assistant.io/blog/2024/07/03/release-20247/

`If you use the Z-Wave JS UI Docker container, you need at least version [9.14.2](https://github.com/zwave-js/zwave-js-ui/releases/tag/v9.14.2).`

Thanks

**Comment by tteck** [comment]
Typing `update` in the LXC console doesn't work?
